hibike/setup.sh

```diff
@@ -9,6 +9,17 @@ cp utils/flash /etc/bash_completion.d/flash.sh
 echo "Setup Tab-complete"
 echo "Please restart your terminals"
 
+sudo usermod -a -G dialout $(whoami)
```

Comment:
This is because we cherry picked some commits from the branch that pr came from. What's the best way to resolve these conflicts now?